packages/kbn-flot-charts/kibana.jsonc

```
@@ -1,5 +1,9 @@
 {
   "type": "shared-common",
   "id": "@kbn/flot-charts",
-  "owner": "@elastic/kibana-operations"
+  "owner": [
+    "@elastic/kibana-presentation",
+    "@elastic/kibana-visualizations",
+    "@elastic/stack-monitoring",
+  ]
```

```suggestion
  "owner": [
    "@elastic/kibana-presentation",
    "@elastic/stack-monitoring",
  ]
```

The visualization team doesn't use this package any more.

---

I pushed https://github.com/elastic/kibana/pull/193351/commits/d68793b7f4706953f5bad98a629f8913881dd678